### ISSUE TITLE
Missing file WorkInProgress.mo

### ISSUE THREAD

**Issue by sjoelund** [kdavies4/FCSys#5]
WorkInProgress.mo is missing from the package, or possibly has been left in the package.order even though it should not:

```
[/var/lib/hudson/slave/workspace/OpenModelica_UPDATE_LIBRARIES/build/FCSys 0.1/package.mo:2:1-1299:10:writable] Error: WorkInProgress was referenced in the package.order file, but was not found in package.mo, WorkInProgress/package.mo or WorkInProgress.mo.
```

**Comment by kdavies4** [comment]
Thanks for the catch.  I have a WorkInProgress subpackage on the development branch, but not on the master branch.  I'll update package.order.

**Comment by kdavies4** [comment]
Fixed with 6143054e86.
